osdk/tests/examples_in_book/write_a_kernel_in_100_lines.rs

```diff
@@ -25,7 +25,12 @@ fn write_a_kernel_in_100_lines() {
 
     // Depends on local OSTD
     let manifest_path = os_dir.join("Cargo.toml");
-    depends_on_local_ostd(manifest_path);
+    depends_on_local_ostd(manifest_path.clone());
+    let tdx_enabled = std::env::var("INTEL_TDX").is_ok();
```

Comment:
The statement `std::env::var("INTEL_TDX").is_ok()` has been repeated so many times. Wrap it into a utility function and add it to the `util` module.